### ISSUE TITLE
Elemental Focus Stone will not proc off of rogue poisons

### ISSUE THREAD

**Issue by Realitycheck1990** [dalaranwow/dalaran-wow#5616]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: I won Elemental Focus Stone(https://www.wowhead.com/item=45866/elemental-focus-stone) today and after beating on a dummy for minutes with poisons on it will not proc the trinket.

**Expected behaviour**: If you read the comments on the item (https://www.wowhead.com/item=45866/elemental-focus-stone) you will see that it is indeed supposed to proc off of rogue poisons, so the trinket should be able to proc off of poisons.

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
